cpu/stm32/ldscripts/stm32.ld

```diff
@@ -34,6 +40,12 @@ SECTIONS
         _sheap2 = ORIGIN(sram4);
         _eheap2 = ORIGIN(sram4) + LENGTH(sram4);
     } > sram4
+
+    .heap4 ALIGN(4) (NOLOAD) :
+    {
+        _sheap3 = ORIGIN(fmcram);
+        _eheap3 = ORIGIN(fmcram) + LENGTH(fmcram);
+    } > fmcram
```

Comment:
I'm not an expert at all with ldscript and I'm wondering if it's possible (and if it makes sense) to force the location of a variable to an external memory using something like:

```c
int var __attribute__((section(".heap4")));
```

or 

```c
int var __attribute__((section(".fmcram")));
```

Reply:
Yes, it would if we change/extend the section definitio
```
    . = ORIGIN(fmcram);
    .fcram.data :
    {
        . = ALIGN(4);
        _fcram_data_start = . ;
        *(.fcram.data .fcram.data.*)
        _fcram_data_end = . ;
    } > fmcram
    
    .fcram.bss :
    {
        . = ALIGN(4);
        _fcram_bss_start = . ;
        *(.fcram.bss .fcram.bss.*)
        _fcram_bss_end = . ;
    } > fmcram
    
    .heap4 ALIGN(4) (NOLOAD) :
    {
        _sheap3 = . ;
        _eheap3 = ORIGIN(fmcram) + LENGTH(fmcram);
    } > fmcram
```

Reply:
Do you think it's worth? I think that for display it can be useful because the declared buffers can be quite large.